tests/app/views/test_brief_response.py

```diff
@@ -326,26 +388,124 @@ def test_day_rate_should_be_less_than_service_max_price(self, live_dos_framework
         assert data == {'error': {'dayRate': 'max_less_than_min'}}
 
 
-class TestSubmitBriefResponse(BaseBriefResponseTest):
+class TestUpdateBriefResponse(BaseBriefResponseTest):
 
-    def _setup_existing_brief_response(self, brief_response_data):
-        res = self.create_brief_response(dict(brief_response_data, **{
+    def setup(self):
+        super(TestUpdateBriefResponse, self).setup()
+        res = self.create_brief_response({
             'briefId': self.brief_id,
             'supplierId': 0,
-        }))
-        assert res.status_code == 201
-
+            'essentialRequirements': [False, False, False, False, False],
+            'niceToHaveRequirements': [False, False, False, False, False],
+            'availability': u'a',
+            'respondToEmailAddress': 'supplier@email.com',
```

Comment:
Would be good to start with a less complete brief response as this is is currently valid (or is it, not sure if it requires a max_day_rate or not to be valid?)